pkg/api/lookout/lookout.proto

```diff
@@ -90,6 +91,7 @@ message GetJobSetsResponse {
 
 message GetJobsRequest {
     string queue = 1;
+    string reason = 10;
```

Comment:
Seeing 10 as the second line trips me up.  

Can we just change this enum so the numbers are increasing? 

```
string reason = 2;
```

I don't really know what the numbers correspond too but I think we are free to change w/e we want in the lookout api.

Reply:
I can move it to the bottom of the list after "9"